scripts/formatChangelog.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
```

Comment:
Since we use `ts-node` and no longer build the script we can now change this and the other one to
```suggestion
#!/usr/bin/env ts-node
```